slacker.py

```diff
@@ -69,7 +69,13 @@ def get_messages_in_time_range(self, oldest, cid, latest=None):
                 murl += "&latest={}".format(latest)
             else:
                 murl += "&latest={}".format(int(time.time()))
-            payload = requests.get(murl).json()
+            response = requests.get(murl)
+            payload = response.json()
+            if payload.get('error') == 'ratelimited':
+                retry_after = int(response.headers['Retry-After'])
+                self.logger.warn('Ratelimited. Sleeping %s', retry_after)
+                time.sleep(retry_after)
+                continue
```

Comment:
@h3h This is the main change from today, where we were getting that KeyError it was due to rate limiting.